homeassistant/components/gios/config_flow.py

```diff
@@ -40,7 +40,7 @@ async def async_step_user(self, user_input=None):
 
                 with timeout(30):
```

Comment:
I'd suggest to store the timeout in a const as well. 
```suggestion
                with timeout(API_TIMEOUT):
```